sensu/plugins/check-ucarp-procs.sh

```diff
@@ -6,7 +6,12 @@ for IFACE in $(ifquery --list); do
       echo "no ucarp process is running for IP ${VIP}"
       exit 2
     fi
-  done
+    for HOSTNAME in `hostname -s`; do
```

Comment:
Simply to avoid setting a variable here for the short hostname.  now that I'm thinking about it, some stacks may report <HOSTNAME>.<LOCATION>, so may need to rework this.